src/main/java/Duke.java

```diff
@@ -1,10 +1,235 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
 public class Duke {
+
+    private final String horizontalLine = "-------------------------";
```

Comment:
You may want the name of your constant variables like this one to match the coding standard. Perhaps something like:
```suggestion
    private final String HORIZONTAL_LINE = "-------------------------";
```